app/src/main/res/layout/activity_main.xml

```diff
@@ -11,7 +11,8 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:text="@string/size_instructions"
-        app:layout_constraintBottom_toTopOf="@id/quantity_parking_slots"
+        android:textSize="35dp"
+        app:layout_constraintBottom_toTopOf="@id/quantity_parking_lots"
         app:layout_constraintLeft_toLeftOf="parent"
```

Comment:
Always use Start/End instead of Left/Right

Reply:
Removed Left/Right and applied Start/End as requested. See example below:
![image](https://user-images.githubusercontent.com/85253636/122767579-319a3e80-d279-11eb-88f6-01f854af7283.png)